modules/EvseManager/tests/ErrorHandlingTest.cpp

```diff
@@ -239,27 +239,32 @@ TEST(ErrorHandlingTest, modify_error_bsp) {
     EXPECT_FALSE(ehs.called_signal_error_cleared);
     EXPECT_FALSE(ehs.called_signal_all_errors_cleared);
 
-    ehs.reset();
-    manager.raise_error("evse_board_support/VendorError", "K2Faults::FAULT_CT_CLAMP",
-                        "Vendor specific error code. Will stop charging session.");
-    EXPECT_FALSE(error_handling.active_errors.all_cleared());
-    EXPECT_TRUE(ehs.signal_error);
-    EXPECT_FALSE(ehs.signal_error_cleared);
-    EXPECT_FALSE(ehs.signal_all_errors_cleared);
-    EXPECT_TRUE(ehs.called_signal_error);
-    EXPECT_FALSE(ehs.called_signal_error_cleared);
-    EXPECT_FALSE(ehs.called_signal_all_errors_cleared);
-
-    ehs.reset();
-    manager.clear_error("evse_board_support/VendorError", "K2Faults::FAULT_CT_CLAMP",
-                        "Vendor specific error code. Will stop charging session.");
-    EXPECT_TRUE(error_handling.active_errors.all_cleared());
-    EXPECT_FALSE(ehs.signal_error);
-    EXPECT_TRUE(ehs.signal_error_cleared);
-    EXPECT_TRUE(ehs.signal_all_errors_cleared);
-    EXPECT_FALSE(ehs.called_signal_error);
-    EXPECT_TRUE(ehs.called_signal_error_cleared);
-    EXPECT_TRUE(ehs.called_signal_all_errors_cleared);
+    // TODO: test raise_error() and clear_error() with error_object
```

Comment:
TODO

Reply:
I would like to keep it, since it is something that is not tested at the moment, but the `EvseManager` module need some rework after merging this anyway, so I didn't want to put to much effort in fixing all test cases

Reply:
ok